### ISSUE TITLE
Implementing the second endpoint of retrieving song details

### ISSUE THREAD

**Issue by magdastone** [NChristina/SAMBA#37]
- [ ] Getting the right results for one song
- [ ] getting the right results for multiple songs (grouped)
- [ ] putting the new structure into the graphs

estimated hrs: 30-40hrs

**Comment by aringot** [comment]
Isn't it the same for one or multiple videos ? (the array can contain one or more, it should not matter)

**Comment by magdastone** [comment]
yeah, but i need to get it working with one first before thinking about multiple at once. this is just a step list for myself :) technically it is the same
